backend/infrahub/core/diff/merger/serializer.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+
+from infrahub.core.constants import DiffAction
+
+from ..model.path import ConflictSelection, EnrichedDiffConflict, EnrichedDiffRoot
+
+
+class DiffMergeSerializer:
```

Comment:
It could be that this is just part of something bigger but in its current state this doesn't look like a class to me. There is no data stored on in the object and it could be cleaner to just have the .serialize() method as a standalone function.

Reply:
this is going to get much bigger as the attributes, relationships, and properties are added